Editor/Window/ConnectToFleetInput.cs

```diff
@@ -0,0 +1,188 @@
+            
+            UpdateGUI();
+        }
+
+        private void RegisterCallBacks(VisualElement container)
+        {
+            container.Q<Button>("ButtonAnywhereConnectButton").RegisterCallback<ClickEvent>(async _ => 
+                await OnAnywhereConnectClicked(_fleetNameInput.text));
+            container.Q<Button>("AnywhereFleetCreateNewButton").RegisterCallback<ClickEvent>(async _ => 
+                await OnCreateNewFleetClicked());
+            _fleetNameDropdownContainer.RegisterValueChangedCallback(evt => OnSelectFleetDropdown());
+            _cancelButton.RegisterCallback<ClickEvent>(_ => OnCancelButtonClicked());
+        }
+
+        private void AssignUiElements(VisualElement container)
+        {
+            _fleetNameInput = container.Q<TextField>("CreateAnywhereFleetField");
+            _fleetNameDropdownContainer = container.Q<DropdownField>("DropdownConnectAnywhereFleet");
+            _fleetId = container.Q("AnywhereFleetID");
+            _fleetIdText = container.Q<Label>("LabelAnywhereConnectedFleetIDValue");
+            _fleetStatus = container.Q("AnywhereFleetStatus");
+            _fleetCreateFoldout = container.Q("FoldoutCreateFleet");
+            _fleetConnectFoldout = container.Q("FoldoutConnectFleet");
+            _cancelButton = container.Q<Button>("AnywhereFleetCancelButton");
+        }
+        
+        private async Task SetupFleetMenu()
+        {
+            await UpdateFleetMenu();
+            _fleetNameDropdownContainer.RegisterValueChangedCallback(_ =>
+                {
+                    _gameLiftPlugin.CurrentState.SelectedFleetIndex = _fleetNameDropdownContainer.index;
+                    var currentFleet = _fleetsList[_gameLiftPlugin.CurrentState.SelectedFleetIndex];
+                    _fleetIdText.text = currentFleet.FleetId;
+                    FleetId = currentFleet.FleetId;
+                    _gameLiftPlugin.CoreApi.PutSetting(SettingsKeys.SelectedFleetIndex,
+                        _gameLiftPlugin.CurrentState.SelectedFleetIndex.ToString());
+                }
+            );
+        }
+        
+        private async Task UpdateFleetMenu()
+        {
+            if (_gameLiftPlugin.GameLiftWrapper != null)
+            {
+                _fleetsList = await _requestAdapter.ListFleets();
+                s_fleetNameList.Clear();
+                _fleetsList.ForEach(fleet => s_fleetNameList.Add(fleet.Name));
+                _fleetNameDropdownContainer.choices = s_fleetNameList;
+            }
+        }
+        
+        private async void SetupBootMenu()
+        {
+            await SetupFleetMenu();
+            
+            if (_fleetsList.Count >= 1)
+            {
+                _fleetState = FleetStatus.Selecting;
+            }
+            _fleetNameDropdownContainer.index = _gameLiftPlugin.CurrentState.SelectedFleetIndex;
+            UpdateGUI();
+        }
+
+        protected sealed override void UpdateGUI()
```

Comment:
Same comment as for RegisterComputeInput

Reply:
Also implemented.